maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -153,6 +153,9 @@ public class Setup extends AbstractServerTask {
 	@Parameter(property = "reuseNodeCache")
 	public Boolean overrideReuseNodeCache;
 
+	@Parameter(property = "spaConfigFile")
+	public String spaConfigFile;
```

Comment:
I think we should revisit the entire set of properties around how O3 is included in the SDK, before we add new properties in.  For example - why do we have `appShellVersion` and `ignorePeerDependencies`, and `overrideReuseNodeCache` here, and should they actually not be here.  And how does the `spaConfigFile` relate to this or not.  @ibacher .  My gut is that all of these should be removed in favor of something more aligned with the rest of the SDK and distributions.